src/store/store.ts

```diff
@@ -1386,6 +1391,9 @@ export const useStore = defineStore('store', {
       this.stateModel.nameRequest = nameRequest
       if (!this.stateModel.tombstone.ignoreChanges) this.stateModel.tombstone.haveUnsavedChanges = true
     },
+    setNameChangedByType (changedByType: boolean) {
+      if (changedByType) this.stateModel.tombstone.nameChangedByType = changedByType
```

Comment:
Why the "if"? Is there no way to set this to False?

Reply:
yeah you're right, updated